uint8 brushLevel
uint8 brushLift
uint8 waterSpray
uint8 wiperLift
uint8 vacuum
bool valve